packages/wrangler/templates/new-worker-scheduled.js

```diff
@@ -4,7 +4,7 @@
  * - Run `wrangler dev --local` in your terminal to start a development server
  * - Run `curl "http://localhost:8787/cdn-cgi/mf/scheduled"` to trigger the scheduled event
  * - Go back to the console to see what your worker has logged
- * - Add a Cron trigger to wrangler.toml (see https://developers.cloudflare.com/workers/wrangler/configuration/#triggers)
+ * - Update the Cron trigger in wrangler.toml
```

Comment:
Out of curiosity, why remove the URL here? I thought that was quite useful?

Reply:
The example cron trigger in the docs link is exactly the same as what this PR adds to wrangler.toml, so I thought I'd remove it.  I can add it it back in though.